backport fix for:
assertion "ref" failed: file "gemrb/plugins/GUIScript/GUIScript.cpp", line 1030,
function "GemRB_View_AddSubview"

from:
https://github.com/gemrb/gemrb/issues/2455

Index: gemrb/CMakeLists.txt
--- gemrb/CMakeLists.txt.orig
+++ gemrb/CMakeLists.txt
@@ -1,6 +1,8 @@
 SET(CMAKE_CXX_VISIBILITY_PRESET "hidden")
 # FreeBSD clang still has symbol visibility issues with the GUIScript plugin
-IF(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" AND CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+IF((CMAKE_SYSTEM_NAME STREQUAL "FreeBSD" OR
+    CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+    AND CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 	SET(CMAKE_CXX_VISIBILITY_PRESET "default")
 ENDIF()
 SET(CMAKE_VISIBILITY_INLINES_HIDDEN TRUE)
